lib/tasks/serve.js

```diff
@@ -172,6 +180,7 @@ exports.shutdown = function (mage, options, cb) {
 	}
 
 	async.series([
+		teardownModules,
 		closeServers,
```

Comment:
I thought the intention was to close servers first, to avoid requests coming in and ending up in our modules. Did your reasoning on this change?